Lib/heapq.py

```diff
@@ -599,9 +599,3 @@ def nlargest(n, iterable, key=None):
     from _heapq import _heappop_max
 except ImportError:
     pass
-
-
-if __name__ == "__main__":
```

Comment:
Please leave this in.  I use it when doing maintenance for the module (we have many such cases in the standard library.